client/src/components/AboutUkraine/AboutUkraine.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React from 'react'
-import {Box, Typography, styled, keyframes  } from '@mui/material';
-// import { useInView } from 'react-intersection-observer';
+import React from 'react';
+import {Box, Typography, styled,   } from '@mui/material';
+import { motion } from 'framer-motion';
```

Comment:
you added a new library but where is the package.json file in the edited files so we could install the dependancy.

Reply:
Done